app/src/main/java/com/phlox/tvwebbrowser/Config.kt

```diff
@@ -42,6 +43,10 @@ class Config(val prefs: SharedPreferences) {
         return prefs.getString(UPDATE_CHANNEL_KEY, "release")!!
     }
 
+    fun isKeepScreenOn(): Boolean {
+        return prefs.getBoolean(KEEP_SCREEN_ON_KEY, false)
```

Comment:
It's Kotlin, feel free to use expression body syntax﻿ :)

``fun isKeepScreenOn() = prefs.getBoolean(KEEP_SCREEN_ON_KEY, false)``

Reply:
Should i also adjust the other lines to keep it consistent?